substratevm/src/com.oracle.svm.core/src/com/oracle/svm/core/c/libc/Libc.java

```diff
@@ -30,7 +30,7 @@
 import java.lang.annotation.Target;
 
 @Retention(RetentionPolicy.RUNTIME)
-@Target({ElementType.METHOD})
+@Target({ElementType.METHOD, ElementType.TYPE})
```

Comment:
You might want to add some more target types. See `org.graalvm.nativeimage.Platforms`

Reply:
I agree, added `ElementType.PACKAGE` to the list of target types.